common/alignment/alignment-fr.html

```diff
@@ -0,0 +1,95 @@
+---
+{
+	"title": "Alignement",
+	"language": "fr",
+	"altLangPage": "alignment-en.html",
+	"dateModified": "2023-06-19"
+}
+---
+<div class="wb-prettify all-pre"></div>
+<h2>Alignement horizontal</h2>
+
+<h3>Aligné à gauche (<code>text-left</code>)</h3>
+<p class="text-left">Texte aligné à gauche.</p>
```

Comment:
As suggested in English, just making this example more obvious and make it distinct from the default left alignment for text.

```suggestion
<div class="text-right">
  <p class="text-left">Texte aligné à gauche.</p>
  <p>Texte aligné à droite</p>
</div>
```